ddtrace/contrib/elasticsearch/patch.py

```diff
@@ -52,39 +52,38 @@ def _perform_request(func, instance, args, kwargs):
         if not pin or not pin.enabled():
             return func(*args, **kwargs)
 
-        with pin.tracer.trace('elasticsearch.query', span_type=SpanTypes.ELASTICSEARCH) as span:
+        with pin.tracer.trace("elasticsearch.query", span_type=SpanTypes.ELASTICSEARCH) as span:
             span.set_tag(SPAN_MEASURED_KEY)
 
             # Don't instrument if the trace is not sampled
             if not span.sampled:
                 return func(*args, **kwargs)
 
             method, url = args
-            params = kwargs.get('params')
-            body = kwargs.get('body')
+            params = kwargs.get("params") or {}
```

Comment:
```suggestion
            params = kwargs.get("params", {})
```
It's subtle but otherwise you'd replace `params=[]` by `{}`, which might would hide the issue.

Reply:
Right, but then if a user explicitly specifies `params=None` then we're still going to get `None` with this suggested change.

Reply:
Oh right. So maybe more something like:

```suggestion
            encoded_params = urlencode(kwargs["params"] if "params" in kwargs else {})
```
Or maybe just don't set the tag at all if `params` is not passed in the args? (if that's possible)

I don't want to nitpick too much on this, I'm just being careful we don't blindly set data that we did not get from the user.

Reply:
I think your snippet has the same issue where if a user specifies `params=None` we're going to use it which results in the bug 😄.

I think we still want to attach the tag, albeit empty, just so that it's explicitly clear to the user that there are no params rather than that we don't report them, or failed to report them. `urlencode` simply results in an empty string for `{}`.

Reply:
> I think we still want to attach the tag, albeit empty, just so that it's explicitly clear to the user that there are no params rather than that we don't report them, or failed to report them. urlencode simply results in an empty string for {}.

Ok, fair enough.